neon_tts_plugin_coqui_ai/__init__.py

```diff
@@ -65,21 +82,63 @@ def get_tts(self, sentence: str, output_file: str, speaker: Optional[dict] = Non
         LOG.debug(to_speak)
         if to_speak:
             with stopwatch:
-                pass
-                # TODO: Get TTS audio here
-
+                wav_data = self.synt.tts(sentence)
```

Comment:
Is it possible to account for the `request_lang` param and load multiple models?

Reply:
I see. I will change it, to init model on `get_tts` stage instead of `__init__`

Reply:
Done with https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/pull/1/commits/d2dfbb16cd089b7930a65c879ad4e3a8d3a56ae5